src/deny-list/scripts.ts

```diff
@@ -0,0 +1,15 @@
+export const checkDenyListScript = `
+  local allDenyListsKey     = KEYS[1]
+  local ipDenyListStatusKey = KEYS[2]
```

Comment:
Maybe we should comment that `allDenyListsKey` explains this key holds:

status -2 for X
status -1 for Y
....

Also, feel free to add some more if needed